c++/src/capnp/capability.h

```diff
@@ -121,6 +121,10 @@ class Request: public Params::Builder {
   RemotePromise<Results> send() KJ_WARN_UNUSED_RESULT;
   // Send the call and return a promise for the results.
 
+  kj::Promise<void> sendIgnore();
```

Comment:
I would name this `sendIgnoringResult()`.

Reply:
👍